### PR TITLE
Refractor

### PR DESCRIPTION
**Pull request by frack113**
some "or" refractor

### CODE REVIEW

File: rules/linux/auditd/lnx_auditd_screencaputre_xwd.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ references:
...
+modified: 2022/12/18
 tags:
     - attack.collection
     - attack.t1113
 logsource:
     product: linux
     service: auditd
 detection:
-    xwd:
+    base:
```
Comment (nasbench) [inline rules/linux/auditd/lnx_auditd_screencaputre_xwd.yml:19]: Use selection for consistency

File: rules/windows/sysmon/sysmon_dcom_iertutil_dll_hijack.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@ detection:
         EventID: 7
         Image|endswith: '\Internet Explorer\iexplore.exe'
         ImageLoaded|endswith: '\Internet Explorer\iertutil.dll'
-    condition: selection_one or selection_two
+    condition: 1 of selection_*
```
Comment (nasbench) [inline rules/windows/sysmon/sysmon_dcom_iertutil_dll_hijack.yml:26]: This rule needs to be split into 2 rules. For `file_event` and `image_load`

**Review by nasbench — COMMENTED** [review]
Also, can we use quotes for values in the `AWS` and `Azure` rules for consistency?

File: rules/windows/sysmon/sysmon_dcom_iertutil_dll_hijack.yml (hunk 2 above)
Comment (nasbench) [reply rules/windows/sysmon/sysmon_dcom_iertutil_dll_hijack.yml:26]: Also, there is another rule that should be split. Take a look at `sysmon_accessing_winapi_in_powershell_credentials_dumping.yml`

File: rules/windows/file/file_event/file_event_win_dcom_iertutil_dll_hijack.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+title: DCOM InternetExplorer.Application Iertutil DLL Hijack - File Event
```
Comment (nasbench) [inline rules/windows/file/file_event/file_event_win_dcom_iertutil_dll_hijack.yml:1]:
```suggestion
title: Potential DCOM InternetExplorer.Application DLL Hijack
```

**Comment by frack113** [comment]
> Also, can we use quotes for values in the `AWS` and `Azure` rules for consistency?

It is done

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Ready for merge

**Pull request merged**